src/overlays/actors/ovl_En_Bomjima/z_en_bomjima.c

```diff
@@ -109,11 +96,15 @@ static AnimationHeader* sAnimations[] = {
     &object_cs_Anim_005DC4, &object_cs_Anim_0026B0, &object_cs_Anim_0036B0, &object_cs_Anim_0031C4,
 };
 
-static u8 D_80C00AE4[] = {
+u8 D_80C00AE4[] = {
     0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x02, 0x00, 0x00,
     0x00, 0x02, 0x00, 0x00, 0x00, 0x02, 0x00, 0x02, 0x00, 0x00,
 };
 
+s16 D_80C00AF8[] = {
+    0x4000, 0x003C, 0x4000, 0x001E, 0xC000, 0x001E, 0xC000, 0x003C,
+};
```

Comment:
A lot of extra 0's in this data as well

Reply:
The first one's an angle, second seems to be a scale, so can go in dec.

Reply:
Sounds good. `D_80C00AF8` really should be either a struct of two s16 elements, or a 2d array but just couldn't get it to match.
As for `D_80C00AE4`, it is actually an array of animmodes so changed it to use the enum